src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,53 @@
+
+## How Sentry Samples
+
+Before understanding how Sentry samples, there are some concepts you need to know:
+
+- Total transaction volume: Total number of transactions the Sentry SDK has observed before any sampling has begun.
+- Client-side(SDK) sampling: The process of selecting which transactions are discarded or sent to Sentry for processing.
+- Server-Side sampling: The process of selecting which transactions are indexed after being processed.
+
+Whether you’re sampling client-side or server-side, it’s always relative to the total transaction volume. Using the first illustration below as an example, we start the process of client-side sampling with a total transaction volume of 4. The client-Side sample rate is set at 50% which leaves only 2 transactions being sent to Sentry for processing. Server-side sampling then takes these 2 processed transactions and indexes only 1 of them because it's sampling at 25% (or a quarter) of the total transaction volume.
+
+![Visualization of how Sentry samples with an effective sampling configuration](server-side-sampling-a.png)
+
+In the next illustration, we show what would happen if the client-side sample rate was lower than the server-side rate. We again start the process of client-side sampling with a total transaction volume of 4. The client-side sample rate is lower this time, set at 25%, leaving only 1 transaction being sent to Sentry for processing. At this stage, you might be asking, “how can server-side sample 50% of the total transaction volume when the client has already discarded 75%?”. It can’t. So even though the server-side sample rate is set to 50% of the total volume, it will only index the 1 transaction it received from the client.
+
+Ultimately, your server-side sample rate must always be equal or lower than your client-side sample rate.
+
+![Visualization of how Sentry samples with an ineffective sampling configuration](server-side-sampling-b.png)
+
+## Traces & Propagation of Sampling Decisions
+
+For Dynamic Sampling, we want to make sampling decisions for entire traces. Sampling transactions using transaction traces is advantageous when trying to understand transactions in the context of the global system. In this case we are not interested in seeing a particular transaction, but the whole group of transactions in a trace, so sampling should occur at the trace level.
+
+When sampling traces, the sampling rules are based on the trace context, this means that the sampling decisions are based on the information extracted from the system that initiated the transaction.
+
+As a mental model: The head transaction in a trace determines the trace context for all following transactions in that trace. No information can be changed, added or deleted after the first propagation. The trace context is bound to only one particular trace, and all the transactions that are part of this trace.
+
```

Comment:
@jan-auer - can you review for accuracy. I've also made some suggestions for edits below: 

```suggestion
When Sentry makes sampling decisions, we do so for entire traces. Instead of looking at a particular transaction, we look at the entire group of transactions in a trace. Sampling transactions using transaction traces is ideal because it allows you to understand transactions in the context of your whole system. 

Since we sample traces, sampling rules are based on the trace context. This means that sampling decisions are based on the information extracted from the _system that initiated the transaction_. The head transaction in a trace determines the trace context for all following transactions in that trace. No information can be changed, added, or deleted after the first propagation. The trace context is bound to only one particular trace, and all the transactions that are part of this trace.

```